src/main/java/com/fasterxml/jackson/databind/deser/impl/PropertyValueBuffer.java

```diff
@@ -261,7 +299,12 @@ public Object handleIdValue(final DeserializationContext ctxt, Object bean) thro
         return bean;
     }
 
-    protected PropertyValue buffered() { return _buffered; }
+    protected PropertyValue buffered() {
+        if (_consumed) {
```

Comment:
This seems like some sort of work-around for something that should not occur... maybe for unwrapped-property handling or something?

Reply:
In `PropertyBasedCreatoer.build(DeserializationContext, PropertyValueBuffer)`, if there's anything buffered, tries to unconditionally tries to call `PropertyValue.assign()` on buffered values. Which simply does not work for parameters. So this check is added.

Reply:
Wrote another way via  https://github.com/FasterXML/jackson-databind/pull/4558/commits/d678cb4fe9947d493691522e7ae09b7af5fa5235, by removing `_consumed` flag. Then instead created new `PropertyValue,setValue()` method and `PropertyValue.AnyParameter` class.

Reply:
Great! I made some further changes, separating existing `_buffered` from new one (difference being old ones go to non-Creator properties; what we need is for "any-setter-for-creator" buffering). And things seem to work well with extended test cases.